social_media_base/models/utm_campaign.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2025 Binhex <https://www.binhex.cloud>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import Command, api, fields, models
+
+
+class UtmCampaign(models.Model):
+    _inherit = "utm.campaign"
+
+    campaign_group_id = fields.Many2one("utm.group.campaign", string="Campaign group")
+    allow_media_ids = fields.Many2many(
+        "social.media", string="Media", compute="_compute_media_id"
+    )
+    media_id = fields.Many2one(
+        "social.media", string="Media", domain="[('id','in',allow_media_ids)]"
```

Comment:
<img width="972" height="86" alt="Image" src="https://github.com/user-attachments/assets/ec661f9f-5c8a-453d-922d-fa148f132fbc" />

```suggestion
        "social.media", string="Allow Media", domain="[('id','in',allow_media_ids)]"
```